homeassistant/components/todoist/calendar.py

```diff
@@ -304,6 +304,19 @@ def __init__(
             str(data[CONF_ID]) if data.get(CONF_ID) is not None else None
         )
 
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        self._apply_coordinator_update()
+        super()._handle_coordinator_update()
+
+    def _apply_coordinator_update(self) -> None:
+        self.data.update()
+        # Set Todoist-specific data that can't easily be grabbed
+        self._cal_data["all_tasks"] = [
```

Comment:
If i understand this right, this can be moved into `extra_state_attributes` so that the entity only needs to depend on `self.data` and these overrides won't be needed.